### ISSUE TITLE
Decompressing a larger file results in daemon communication issue

### ISSUE THREAD

**Issue by vanjmali** [wisp-gg/frontend#320]
### Describe the bug

While doing 3 dots (`...`) > `Decompress` on an compressed file, such as WISP's generated `.tar.gz` or `.zip` that what we believe is over 100MB throws an `There was an exception while attempting to communicate with the daemon.` error.
Console log:
```
Uncaught (in promise) Error: TranslatableError: generic.daemon_connection_exception
```
No logs found in WISP log.

### Expected Behavior

Decompress the file normally.

### Actual Behavior

Throws `Uncaught (in promise) Error: TranslatableError: generic.daemon_connection_exception`

### Steps to Reproduce

1. Open File Manager
2. Create or upload an compressed file that is large
3. Use 3 dots on it and do `Decompress`

### Browser Details

Microsoft Edge Version 104.0.1293.70 (Official build) (64-bit)

### Operating System

Windows 11 Pro

**Comment by victiondev** [comment]
same issue as https://github.com/wisp-gg/frontend/issues/308 - issue is fixed but not yet pushed to production, will likely be pushed tomorrow

**Comment by vanjmali** [comment]
Action was not completed to opposite of #308 and doesn't appear to be just a timeout, the reason why we've created a separate issue. If this is also solved by #308 then it's good to go.
